packages/ui/src/utils/env.ts

```diff
@@ -1 +1 @@
-export const __DEV__ = typeof process !== 'undefined' && process.env.NODE_ENV !== 'production'
+export const __DEV__ = typeof process === 'undefined' ?? process.env.NODE_ENV !== 'production'
```

Comment:
Check this: https://github.com/unjs/std-env, maybe we could use it instead.

Reply:
`typeof process === 'undefined' ` will always return Boolean which is not undefined or null, so `??` will return left operand always...